lib/correios/cep/web_service.rb

```diff
@@ -48,20 +44,31 @@ def build_request(zipcode)
         request = Net::HTTP::Post.new(uri.path)
         request['Content-Type'] = 'text/xml; charset=utf-8'
         request.body = request_body(zipcode)
+        Correios::CEP.log_request(request, uri.to_s)
```

Comment:
Not sure to log request implicitly here. This method supposed only to build a request.